src/pages/en/guides/deploy.md

```diff
@@ -213,6 +213,16 @@ Create a new `netlify.toml` file at the top level of your project repository wit
   publish = "dist"
 ```
 
+If you are using `pnpm`, then set the following settings instead:
```

Comment:
```suggestion
Using `pnpm`? [Netlify needs some extra set-up to support `pnpm`.](https://answers.netlify.com/t/using-pnpm-and-pnpm-workspaces/2759) We suggest starting with the following settings in your `netlify.toml`:
```

Reply:
```suggestion
Using [`pnpm` with Netlify?](https://answers.netlify.com/t/using-pnpm-and-pnpm-workspaces/2759) Use the following settings instead:
```

Reply:
✂️💨 Scissors to the rescue! LGTM.

Reply:
Ah, good question. I think I’d say “on”?

Reply:
Updated in my suggestion, thanks! @RafidMuhymin can make the final call.  :)